modules/core/src/test/scala/io/janstenpickle/trace4cats/SpanSpec.scala

```diff
@@ -18,10 +18,11 @@ import scala.concurrent.duration._
 
 class SpanSpec extends AnyFlatSpec with Matchers with ScalaCheckDrivenPropertyChecks with ArbitraryInstances {
   val ec: TestContext = TestContext()
-  implicit val timer: Timer[IO] = ec.ioTimer
-  implicit val ctx: ContextShift[IO] = ec.ioContextShift
-
-  val sc: ScheduledExecutorService = new ScheduledThreadPoolExecutor(1)
+  implicit val ioRuntime: IORuntime = {
```

Comment:
This actually hangs for eternity. I think we should migrate to a framework that allows writing tests without calling `unsafeRunSync()`. Will do it later. FS2 uses munit.